### PR TITLE
Implementing character data (health, damage).

### PR DESCRIPTION
**Pull request by predragspanovic-playrix**

### CODE REVIEW

File: src/CharacterData.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+
+class CharacterData {
+private:
```
Comment (dusanslivka-playrix) [inline src/CharacterData.h]: Treba prvo da ide **public** pa onda **private**. [coding standards](https://knowledge.playrix.com/display/development/Coding+Standards#CodingStandards-%D0%9F%D0%BE%D1%80%D1%8F%D0%B4%D0%BE%D0%BA%D0%BE%D0%B1%D1%8A%D1%8F%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F)

File: src/CharacterData.h [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	int GetHealth() const; 
+	int GetDamage() const;
+
+	void SetHealth(int health);
+	void SetDamage(int damage);
+
+	void AddHealth(int health);
+	void AddDamage(int damage);
+
+	void SubHealth(int health);
```
Comment (dusanslivka-playrix) [inline src/CharacterData.h]: Trebalo bi neko smislenije ime, recimo **DecreaseHealth**.

File: src/CharacterData.h [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+	void FullHealth();
```
Comment (dusanslivka-playrix) [inline src/CharacterData.h]: Isto treba neko smislenije ime posto metoda vraca health na maksimum.

File: src/CharacterData.cpp [hunk 4]
```diff
@@ -0,0 +1,55 @@
+#include "CharacterData.h"
+
+
+
+
+
+int CharacterData::GetHealth() const
```
Comment (oliverabikickekenj-playrix) [inline src/CharacterData.cpp]: Getere treba definisati header file-u(dokumentacija zahteva).

Comment (leventesimonplayrix) [reply src/CharacterData.cpp]: Zar nije da pise da moze a ne mora?

**Making the code in the header files**
The header file is read more often than * .cpp. Therefore, it is desirable that it be compact. If you _decide_ to put a function definition in the header file, then you should strive to take as little space as possible. If it turns out, then 1 line (and if the function is large, then think once again whether it should be inline exactly). 

>if you decide

Comment (oliverabikickekenj-playrix) [reply src/CharacterData.cpp]: Dalje u dokumenatciji je navedeno .." the definitions of the getter functions should be placed in the header file, so we simplify the work of the optimizer"..
should be -> treba

Comment (leventesimonplayrix) [reply src/CharacterData.cpp]: True enough.

File: src/CharacterData.h [hunk 5]
```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+
+class CharacterData {
+
+public:
+	
+	int GetHealth() const
```
Comment (igorkukobat-playrix) [inline src/CharacterData.h]: preglednije da stavis sve u jednu liniju 
int GetDamage() const { return _damage; }

File: src/CharacterData.h [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+	void RestoreFullHealth();
+
+	
+private:
+
+	int _health;  //load data from save
+	int _damage;  //load data from save
+
+	const int maxHealth;  //load from save
```
Comment (igorkukobat-playrix) [inline src/CharacterData.h]: donja crta fali, i nije mi jasno kako ce se ona popunjavati, posto jos nije nigde to uradjeno. Hoce ici iz konfiguracionog fajla ?

**Pull request merged**